app/src/main/java/one/mixin/android/vo/QuoteMessageItem.kt

```diff
@@ -3,70 +3,41 @@ package one.mixin.android.vo
 import android.annotation.SuppressLint
 import android.os.Parcelable
 import androidx.room.Entity
-import com.google.gson.annotations.SerializedName
 import kotlinx.parcelize.Parcelize
-import one.mixin.android.util.GsonHelper
+import one.mixin.android.moshi.MoshiHelper.getTypeAdapter
 
 @SuppressLint("ParcelCreator")
 @Entity
 @Parcelize
 data class QuoteMessageItem(
```

Comment:
[QuoteMessageItemJsonAdapter](https://github.com/MixinNetwork/android-app/commit/e022580f9cdae5ed07ec73a0aeb56c2670e1f419#diff-e05bd4ecee8a589fdc93f58dc2825145e3fe31daddcbf8405d120e65a0128501R14), it supports alternate